docs/HISTORY.txt

```diff
@@ -5,7 +5,10 @@ Changelog
 2.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Update sliderpane for accessiblity
+  - Show always empty image alt and title attributes on a pane
+  - Remove unused descriptionfield on a pane
+  [elioschmutz]
```

Comment:
:+1: you're right... now it should be good:

![bildschirmfoto 2015-10-30 um 10 08 18](https://cloud.githubusercontent.com/assets/557005/10841955/39129926-7eee-11e5-9c38-5fcdbffde340.png)